packages/react-components/react-toast/src/state/useToaster.ts

```diff
@@ -38,17 +38,17 @@ export function useToaster<TElement extends HTMLElement = HTMLDivElement>(option
   }, []);
 
   const pauseAllToasts = React.useCallback(() => {
-    for (const toastId of toaster.visibleToasts) {
+    toaster.visibleToasts.forEach(toastId => {
       const toast = toaster.toasts.get(toastId);
       toast?.imperativeRef.current?.pause();
-    }
+    });
   }, [toaster]);
 
   const playAllToasts = React.useCallback(() => {
-    for (const toastId of toaster.visibleToasts) {
+    toaster.visibleToasts.forEach(toastId => {
```

Comment:
had to make this change because @fluentui/react-portal-compat has ES5 as a target and there were TS compilation issues with interating sets/iterators directly